4_arm.ipynb

```diff
@@ -481,11 +836,78 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's use a function called move _wholebody_ ik that summarizes the above processing. The arguments are the 3D position and the rotation angle of the hand.\n",
```

Comment:
There was an extra space in "_wholebody_ ik", so I corrected it to be "_wholebody_ik".  See the following:

    "Let's use a function called move _wholebody_ik that summarizes the above processing. The arguments are the 3D position and the rotation angle of the hand.\n",

Reply:
"move_wholebody_ik" is correct. Google Translate sometimes chunks word with underscore... (and I missed to correct them)